Description: relax dependencies to match versions in Debian
Author: Emanuele Rocca <ema@debian.org>
Last-Update: 2023-01-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: psa-crypto-sys/Cargo.toml
===================================================================
--- psa-crypto-sys.orig/Cargo.toml
+++ psa-crypto-sys/Cargo.toml
@@ -31,7 +31,7 @@ license = "Apache-2.0"
 repository = "https://github.com/parallaxsecond/rust-psa-crypto"
 
 [build-dependencies.bindgen]
-version = "0.59.2"
+version = "0.66"
 optional = true
 
 [build-dependencies.cc]
